PWGHF/D2H/Tasks/taskD0.cxx

```diff
@@ -26,6 +26,7 @@ using namespace o2;
 using namespace o2::framework;
 using namespace o2::framework::expressions;
 using namespace o2::aod::hf_cand_2prong;
+using namespace o2::aod::hf_cand;
```

Comment:
Where do you use this?

Reply:
This is used for the` VertexerType::DCAFitter` and `VertexerType::KfParticle`